tests/testthat/test_XcmsExperimentHdf5-functions.R

```diff
@@ -0,0 +1,1083 @@
+library(rhdf5)
```

Comment:
should the library() be in the parent directory, for all *hdf5 tests  ?

Reply:
I wanted to keep the HDF5 functionality as separate as possible thus I did not add it into the main *testthat.R* file. I find it *cleaner* if the unit test file specifically sets the stage for its R source file.